src/include/duckdb/storage/compression/alp/alp.hpp

```diff
@@ -0,0 +1,31 @@
+//===----------------------------------------------------------------------===//
+//                         DuckDB
+//
+// duckdb/storage/compression/alp/alp.hpp
+//
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "duckdb/storage/compression/alp/algorithm/alp.hpp"
+#include "duckdb/common/assert.hpp"
+#include "duckdb/common/exception.hpp"
+#include "duckdb/common/helper.hpp"
+#include "duckdb/common/limits.hpp"
+#include "duckdb/function/compression_function.hpp"
+
+namespace duckdb {
+
+template <class T, bool EMPTY>
+struct AlpState {
+public:
+	using EXACT_TYPE = typename FloatingToExact<T>::type;
+	AlpState(void *state_p = nullptr) : data_ptr(state_p), alp_state() {
+	}
+	//! The Compress/Analyze State
+	void *data_ptr;
```

Comment:
True. This was a leftover from the patas code. But since we don’t really do anything here; I will better remove this class and directly use the `alp::AlpCompressionState` class for the state.